src/view/InputBox.tsx

```diff
@@ -19,11 +12,16 @@ interface InputBoxProps {
  * InputBox is where the user can enter their boolean expression.
  */
 export const InputBox = React.memo(function InputBox(props: InputBoxProps) {
-  const { query } = props;
-
-  const onChange = (event: React.ChangeEvent<HTMLInputElement>) => {
-    props.onQueryChange(event.target.value);
+  const codeMirrorOptions = {
```

Comment:
could pull this const out

also will python syntax highlighting work for some of our operators that don't exist in python?

Reply:
ah i see the custom highlighting checkbox in the description

Reply:
Oh was gonna leave a comment asking - what's the React way of setting this value on initial render?  With this current code, we duplicate this default `p and q` value upstream

Reply:
i think we could use props.query? since that's being set on initial render upstream

Reply:
and you can do it on initial render with a useEffect hook and passing empty list as the second parameter